numpy/lib/tests/test_function_base.py

```diff
@@ -1756,7 +1756,24 @@ def test_simple(self):
         assert_array_equal(unwrap([1, 1 + 2 * np.pi]), [1, 1])
         # check that unwrap maintains continuity
         assert_(np.all(diff(unwrap(rand(10) * 100)) < np.pi))
-
+    
+    def test_period(self):
+        # check that unwrap removes jumps greater that 255
+        assert_array_equal(unwrap([1, 1 + 256], period=255), [1, 2])
+        # check that unwrap maintains continuity
+        assert_(np.all(diff(unwrap(rand(10) * 1000, period=255)) < 255))
```

Comment:
Isn't this diff allowed to be `= 255`?

Reply:
This is a test I copy-and-pasted from the old PR and just adopted it to the new parameter names. I have to say, I don't really like it. Let's change that maybe.

My understanding was that this test should ensure that the correction is not too large, e.g. if a correction of 4 pi is needed, there should not be a correction of 6 pi, or so. But not so sure.
Regarding your question of the diff being `=255`, I think, that should not be the case. Let's say the period is `2 pi`, after unwrapping there must not be a jump larger than `pi`. So jumps of `2 pi` are not allowed.